bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v1/SearchImplTest.java

```diff
@@ -47,6 +54,21 @@ void testSearchProperties() {
         assertEquals(3, search.getSearchTermMinimumLength());
         assertEquals("/jcr:content/search", search.getRelativePath());
         assertEquals("core/wcm/components/search/v1/search", search.getExportedType());
+        assertEquals("/content/en/search", search.getSearchRootPagePath());
+        assertSame(search.getSearchRootPagePath(), search.getSearchRootPagePath());
```

Comment:
Why are you testing that both objects refer to the same object?

Reply:
It was just a test that it doesn't recompute the object.
Subsequent calls follow different branches, so this was mostly just crafting tests to make the branch coverage complete.